staker-rewards/rewards.js

```diff
@@ -0,0 +1,128 @@
+import clc from "cli-color"
+
+import Context from "./lib/context.js"
+import SLACalculator from "./lib/sla-calculator.js"
+
+async function run() {
+    try {
+        // URL to the websocket endpoint of the Ethereum node.
+        const ethHostname = process.env.ETH_HOSTNAME
+        // Start of the interval passed as UNIX timestamp.
+        const intervalStart = process.argv[2]
+        // End of the interval passed as UNIX timestamp.
+        const intervalEnd = process.argv[3]
+        // Determines whether debug logs should be disabled.
+        const isDebugDisabled = process.env.DEBUG !== "on"
+        // Determines whether the cache refresh process should be performed.
```

Comment:
We should add a warning that this option is only for developers and one should disable cache refresh only when they really know what they are doing. Also, that not performing cache refresh may cause rewards distribution to be calculated based on outdated information from the chain.